### PR TITLE
Brease pet backend

### PR DESCRIPTION
**Pull request by nour-d**

### CODE REVIEW

File: src/vetportal/DashboardsGui.java [hunk 1]
```diff
@@ -47,77 +45,287 @@ public DashboardsGui(VetPortal vetPortal) throws ParseException {
...
+
+        pClientField.setFont(new java.awt.Font("Calibri", 0, 14)); // NOI18N
+
+        pNameField.setFont(new java.awt.Font("Calibri", 0, 14)); // NOI18N
+
+        petTableScroll.setBackground(new java.awt.Color(255, 255, 255));
+
+        petsTable.setFont(new java.awt.Font("Calibri", 0, 14)); // NOI18N
+        petsTable.setModel(new javax.swing.table.DefaultTableModel(
+                new Object [][] {
```
Comment (BekahT) [inline src/vetportal/DashboardsGui.java]: Don't add empty rows to the table upon load. Just pass an empty object. See Clients table.

File: src/vetportal/DashboardsGui.java [hunk 2]
```diff
@@ -47,77 +45,287 @@ public DashboardsGui(VetPortal vetPortal) throws ParseException {
...
+
+        aPetSearch.setFont(new java.awt.Font("Calibri", 0, 14)); // NOI18N
+        aPetSearch.setText("Pet:");
+
+        appointmentTableScroll.setBackground(new java.awt.Color(255, 255, 255));
+
+        appointmentsTable.setFont(new java.awt.Font("Calibri", 0, 14)); // NOI18N
+        appointmentsTable.setModel(new javax.swing.table.DefaultTableModel(
+                new Object [][] {
+                        {null, null, null, null, null, null},
```
Comment (BekahT) [inline src/vetportal/DashboardsGui.java]: Don't add empty rows to the table on load. See Clients table.

File: src/vetportal/GuiBuilderForms/DashboardsGui.java [hunk 3]
```diff
@@ -0,0 +1,534 @@
+/*
```
Comment (BekahT) [inline src/vetportal/GuiBuilderForms/DashboardsGui.java]: We should probably delete this file it its contents were moved into the DashboardsGui.java file that is not in the GuiBuilderForms folder.

**Review by BekahT — CHANGES_REQUESTED** [review]

**Review by nour-d — COMMENTED** [review]
Accepted the changes.

File: src/vetportal/DashboardsGui.java (hunk 1 above)
Comment (nour-d) [reply src/vetportal/DashboardsGui.java]: Fixed

File: src/vetportal/GuiBuilderForms/DashboardsGui.java (hunk 3 above)
Comment (nour-d) [reply src/vetportal/GuiBuilderForms/DashboardsGui.java]: Fixed

**Review by BekahT — APPROVED** [review]

**Pull request merged**